adapters/triplelift_native/triplelift_native.go

```diff
@@ -66,32 +60,32 @@ func processImp(imp *openrtb2.Imp, request *openrtb2.BidRequest) error {
 	if imp.Native == nil {
 		return fmt.Errorf("no native object specified")
 	}
-	if tlext.InvCode == "" {
-		return fmt.Errorf("no inv_code specified")
-	}
```

Comment:
This error check isn't needed because `InvCode` is a required parameter as defined in `static/bidder-params/triplelift_native.json` so if it is missing an error will occur upstream and this adapter will never run.

Reply:
The json schema required check does not duplicate this functionality. An empty string satisfies the required check. If you want to move this check to the json schema, you need to define a minimum length of 1. This is technically a breaking change, but likely fine since the placement would fail with this error.

Reply:
👍 I added minimum length of 1 to the JSON schema and also added `params_test.go` which was missing.